### ISSUE TITLE
Failing to install

### ISSUE THREAD

**Issue by pH142857** [doug-101/TreeLine#27]
I didn't manage to properly install and run Treeline on my Debian. I tried to follow the scarce instructions on https://treeline.bellz.org/install.html:

    python3 install.py
But I needed to install python, so I did:

    sudo apt-get install python3
    python3 install.py
But it was missing PyQt, so I looked it up and tried to follow the instructions in https://doc.qt.io/qtforpython-6/. I saw there were `pip` command so I looked up how to install `pip`:

    sudo apt install python3-pip
Then:

    python3 -m venv env
But I had to install venv so:

    apt install python3.11-venv
    python3 -m venv env
Then:

    source env/bin/activate
Which changed my terminal environment as far as I understand, then:

    pip install pyside6
and finally:

    python3 install.py
But, halas! `PyQt not found`
So after a new search I guessed PyQt was equivalent to pyside6 but the install script didn't know it, so I tried to skip dependency checks, as you mention: `python install.py -x`

Treeline seems to finally install and appears in the apps but it fails to launch.

So, long story short, I dug quite far in a programming environment I don't know just to install a piece of software, it was troublesome and it didn't work. Could you provide a step-by-step guide on how to install Treeline on Linux please?

**Comment by doug-101** [comment]
I think your primary problem was using pip and venv, which installs into a virtual python environment, making things more complex, as well as missing some requirements.  I recommend using native Debian packages for the requirements listed on [my page](https://treeline.bellz.org/require.html).

But, assuming you're running Debian 12 bookworm or newer, the easiest, single-step installation is:  

> sudo apt install treeline

**Comment by pH142857** [comment]
It works, thanks.
